### PR TITLE
ADD BeanEco SmartChain

### PR DESCRIPTION
**Pull request by briannichols0702**
{
name": "BeanEco SmartChain",
  "chain": "BESC",
  "rpc": ["https://www.mainnet-rpc.bescscan.io],
  },
  "faucets": [https://faucet.bescscan.io/],
  "nativeCurrency": {
    "name": "BeanEco SmartChain",
    "symbol": "BESC",
    "decimals": 18
  },
  "infoURL": "https://besceco.finance",
  "shortName": "BESC",
  "chainId": 535037,
  "networkId":535037,

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
Your filename is completely wrong

**Comment by briannichols0702** [comment]
> Your filename is completely wrong

Ahh sorry about that I believe I got it fixed.

**Comment by ligi** [comment]
a bit better - still very wrong though

**Review by Imebeez — APPROVED** [review]

**Review by ligi — CHANGES_REQUESTED** [review]
nope - it really looks like you have no idea what you are doing - can someone else of your team take over? I kinda don't want to spend time reviewing your PRs for free anymore

**Comment by briannichols0702** [comment]
> nope - it really looks like you have no idea what you are doing - can someone else of your team take over? I kinda don't want to spend time reviewing your PRs for free anymore



> nope - it really looks like you have no idea what you are doing - can someone else of your team take over? I kinda don't want to spend time reviewing your PRs for free anymore

I apologize Send me an ur Busd address I’ll make it worth ur time 😂 was trying to
Get this done while
My dev has weekend
Off.

**Comment by ligi** [comment]
send something to lists.eth and I fix up the PR

**Comment by briannichols0702** [comment]
> 

Sponsored you with lil something for your help

File: _data/chains/eip155-535037.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+   "name": "BeanEco SmartChain",
+   "title": "BESC Mainnet",
+   "chain": "BESC",
+   "rpc": ["mainnet-rpc.bescscan.io", "thegraph.bescscan.io"],
```
Comment (ligi) [inline _data/chains/eip155-535037.json:5]:
```suggestion
   "rpc": ["https://mainnet-rpc.bescscan.io"],
```

**Review by ligi — APPROVED** [review]

**Review by Imebeez — COMMENTED** [review]
👍

**Review by Imebeez — COMMENTED** [review]
👍

**Pull request merged**